### PR TITLE
update xpdstack

### PR DESCRIPTION
**Pull request by CJ-Wright**
If this could go in quickly that would be great!

### CODE REVIEW

**Comment by CJ-Wright** [comment]
Ready for review!

**Comment by CJ-Wright** [comment]
@mrakitin would it be possible for me to get some review/merge?

File: recipes-tag/streamz_ext/meta.yaml [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+{% set sha256 = "e8054418130ecb28272181f626f0c65d7596be73462f97ffa2e584ee75720b49" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://github.com/xpdAcq/{{ name }}/archive/{{ version }}.tar.gz
+  url: https://github.com/xpdAcq/{{ name }}/releases/download/{{ version }}/{{ version }}.tar.gz
```
Comment (mrakitin) [inline recipes-tag/streamz_ext/meta.yaml]: What's the difference between the `archive` and the `releases/download` here? Is it something new by GitHub?

File: recipes-tag/xpdacq/meta.yaml [hunk 2]
```diff
@@ -34,7 +34,7 @@ requirements:
     - databroker
     - ophyd
     - xpdconf
-
+
```
Comment (mrakitin) [inline recipes-tag/xpdacq/meta.yaml]: I think this updates are not needed, if possible.

File: recipes-tag/xpdacq/meta.yaml [hunk 3]
```diff
@@ -45,7 +45,7 @@ test:
 about:
   home: https://github.com/xpdAcq/xpdAcq
   license: BSD-3-Clause
-  license_family: BSD
+  license_family: BSD 
   license_file: LICENSE
```
Comment (mrakitin) [inline recipes-tag/xpdacq/meta.yaml]: See above.

File: recipes-tag/xpdacq/meta.yaml [hunk 4]
```diff
@@ -54,4 +54,4 @@ about:
 
 extra:
   recipe-maintainers:
-    - CJ-Wright
+    - CJ-Wright
```
Comment (mrakitin) [inline recipes-tag/xpdacq/meta.yaml]: See above.

File: recipes-tag/xpdacq/meta.yaml [hunk 5]
```diff
@@ -13,7 +13,7 @@ source:
 
 build:
   noarch: python
-  number: 1
+  number: 0
```
Comment (mrakitin) [inline recipes-tag/xpdacq/meta.yaml]: Should the corresponding version be updated as well? Or is it still `0.8.1`?

Comment (CJ-Wright) [reply recipes-tag/xpdacq/meta.yaml]: Ah maybe this should be left off, I'm not certain why we had a build number bump on your side, but I'll remove it.

File: recipes-tag/streamz_ext/meta.yaml (hunk 1 above)
Comment (mrakitin) [reply recipes-tag/streamz_ext/meta.yaml]: Oh, good to know, thanks!

**Comment by CJ-Wright** [comment]
Thanks!

**Pull request merged**